tests/RPCClient.test.ts

```diff
@@ -1213,60 +1140,53 @@ describe(`${RPCClient.name}`, () => {
         });
-
         await rpcClient.duplexStreamCaller<
           JSONRPCRequestParams,
           JSONRPCResponseResult
         >(methodName, {
           timer: lowerTimeoutTime,
         });
-
         const ctx = await ctxP;
         expect(ctx.timer.delay).toBe(lowerTimeoutTime);
         ctx.timer.cancel();
         await ctx.timer.catch(() => {});
       },
     );
-    test.prop({
-      timeouts: rpcTestUtils.timeoutsArb,
-    })(
-      'check that call with ctx can override lower timeout of RPCClient',
-      async ({ timeouts: [lowerTimeoutTime, higherTimeoutTime] }) => {
-        const streamPair: RPCStream<Uint8Array, Uint8Array> = {
-          cancel: () => {},
-          meta: undefined,
-          readable: new ReadableStream(),
-          writable: new WritableStream(),
-        };
-        const { p: ctxP, resolveP: resolveCtxP } = promise<ContextTimed>();
-        const rpcClient = new RPCClient({
-          manifest: {},
-          streamFactory: async (ctx) => {
-            resolveCtxP(ctx);
-            return streamPair;
-          },
-          logger,
-          idGen,
-          timeoutTime: lowerTimeoutTime,
-        });
-
-        await rpcClient.duplexStreamCaller<
-          JSONRPCRequestParams,
-          JSONRPCResponseResult
-        >(methodName, {
-          timer: higherTimeoutTime,
-        });
-
-        const ctx = await ctxP;
-        expect(ctx.timer.delay).toBe(higherTimeoutTime);
-        ctx.timer.cancel();
-        await ctx.timer.catch(() => {});
-      },
-    );
-    test.prop({
-      timeoutTime: fc.integer({ min: 0 }),
-    })(
-      'check that call with ctx can override lower timeout of RPCClient with Infinity',
+    // Disabled to get feedback.
+    // test.prop({ timeouts: rpcTestUtils.timeoutsArb })(
+    //   'calling with ctx should override lower timeout of RPCClient',
+    //   async ({ timeouts: [lowerTimeoutTime, higherTimeoutTime] }) => {
+    //     const streamPair: RPCStream<Uint8Array, Uint8Array> = {
+    //       cancel: () => {},
+    //       meta: undefined,
+    //       readable: new ReadableStream(),
+    //       writable: new WritableStream(),
+    //     };
+    //     const { p: ctxP, resolveP: resolveCtxP } = promise<ContextTimed>();
+    //     const rpcClient = new RPCClient({
+    //       manifest: {},
+    //       streamFactory: async (ctx) => {
+    //         resolveCtxP(ctx);
+    //         return streamPair;
+    //       },
+    //       logger,
+    //       idGen,
+    //       timeoutTime: lowerTimeoutTime,
+    //     });
+    //     await rpcClient.duplexStreamCaller<
+    //       JSONRPCRequestParams,
+    //       JSONRPCResponseResult
+    //     >(methodName, {
+    //       timer: higherTimeoutTime,
+    //     });
+    //     const ctx = await ctxP;
+    //     expect(ctx.timer.delay).toBe(higherTimeoutTime);
+    //     ctx.timer.cancel();
+    //     await ctx.timer.catch(() => {});
+    //   },
+    // );
```

Comment:
Do we need to check if the timeout can override both lower and higher timeouts? Shouldn't override be like `originalVal = override` and not matter if the value was higher or lower?

Reply:
Okay, so the behaviour between the commands are different, so we need both the tests to exist.